src/modules/dashboard/components/Dialogs/ControlSafeDialog/SafeTransactionPreview.tsx

```diff
@@ -234,6 +264,42 @@ const SafeTransactionPreview = ({ colony, values }: Props) => {
     values.transactions[index].transactionType ===
     TransactionTypes.TRANSFER_NFT;
 
+  const getDetailsItemValue = (input, transaction) => {
+    switch (true) {
+      case input.type === 'address': {
+        return <MaskedAddress address={transaction[input.name]} />;
+      }
+      case input.type === 'address[]': {
+        const formattedArray = getArrayFromString(transaction[input.name]);
+
+        const maskedArray = formattedArray.map((address, index, arr) => {
+          return (
+            <div>
+              <MaskedAddress address={address} />
```

Comment:
Consider passing `address={address.trim()}` here to avoid "Address format is wrong!":

![address-wrong](https://user-images.githubusercontent.com/64402732/194064114-c00f1e61-0c69-452c-bb18-801a17e87250.png)

Reply:
Sounds good!